data/json/items/armor.json

```diff
@@ -5146,94 +5146,77 @@
     "flags": [ "VARSIZE" ]
   },
   {
-    "id": "judo_belt_black",
+    "abstract": "judo_belt_abstract",
     "type": "ARMOR",
-    "name": "black belt",
-    "description": "A belt worn by martial arts practitioners, advertising a master.",
+    "name": "judo belt template",
+    "description": "This is a template for judo belts. If found in a game it is a bug.",
     "weight": 200,
     "volume": 1,
     "price": 1000,
     "to_hit": -1,
     "material": [ "cotton" ],
     "symbol": "[",
+    "color": "pink",
```

Comment:
The abstract definition shouldn't have any elements that the individual instances override, in this case color.

Reply:
Ok. I thought it needed a default value. Should I remove the description as well?